base/bucket_test.go

```diff
@@ -302,14 +305,23 @@ func TestGetViewQueryTimeout(t *testing.T) {
 	assert.Equal(t, expectedViewQueryTimeout, fakeBucketSpec.GetViewQueryTimeout())
 }
 
-var (
-	rootKeyPath    = filepath.Join(os.TempDir(), "root.key")
-	rootCertPath   = filepath.Join(os.TempDir(), "root.pem")
-	clientKeyPath  = filepath.Join(os.TempDir(), "client.key")
-	clientCertPath = filepath.Join(os.TempDir(), "client.pem")
-)
+func mockCertificatesAndKeys(t *testing.T) map[string]string {
```

Comment:
Returning a map of file paths here feels a little bit hard to follow. The consumer of this has to know the internal detail of this function (the map key names) in order to get those values back out.

Can we return a bunch of named strings instead?

```suggestion
func mockCertificatesAndKeys(t *testing.T) (certPath, clientCertPath, clientKeyPath, rootCertPath, rootKeyPath string) {
```

Reply:
Makes sense. This has been fixed in latest revision.